### ISSUE TITLE
【FR】Bug: addmm/bmm/mm/rms_norm/softmax performance degradation on huawei (LLM-Research/Llama-3.2-1B-Instruct)

### ISSUE THREAD

**Issue by Lxiparer** [FlagOpen/FlagGems#4551]
### Description

During performance benchmarking on model **LLM-Research/Llama-3.2-1B-Instruct**, the following operators were identified as causing significant performance degradation (<80% of native baseline):

- `addmm`
- `bmm`
- `mm`
- `rms_norm`
- `softmax`

### Environment

| Item | Value |
|------|-------|
| Hardware | huawei |
| GPU | Ascend910 |
| PyTorch | 2.8.0+cpu |
| FlagGems | 5.0.2 |
| vLLM | 0.13.0 |

### Steps to Reproduce

1. Set up environment on huawei
2. Install dependencies (PyTorch + Triton + FlagGems + vLLM)
3. Run performance benchmark with V1 (native) and V2 (FlagGems enabled)
4. Compare V1 vs V2 throughput, ratio below 80% at specific concurrency levels
5. Run operator search optimization to identify problematic operators

### Expected Behavior

All FlagGems operators should maintain performance at >=80% of native baseline at each concurrency level.

### Actual Behavior

The following operators cause performance degradation:
- `addmm`: performance degradation detected
- `bmm`: performance degradation detected
- `mm`: performance degradation detected
- `rms_norm`: performance degradation detected
- `softmax`: performance degradation detected

These operators have been disabled to achieve acceptable performance.

### FlagGems Integration Code

**File**: `/vllm-workspace/vllm-ascend/vllm_ascend/worker/model_runner_v1.py`
**Enable call**: `flag_gems.enable(record=True, once=True, path="/root/gems.txt")`

**Current gems.txt** (9 operators, path: `/root/gems.txt`):
```
[DEBUG] flag_gems.runtime._ascend.ops.amax.amax: GEMS_ASCEND AMAX
[DEBUG] flag_gems.runtime._ascend.ops.ones.ones: GEMS_ASCEND ONES
[DEBUG] flag_gems.runtime._ascend.ops.zeros.zeros: GEMS_ASCEND ZEROS
[DEBUG] flag_gems.runtime._ascend.ops.arange.arange_start: GEMS_ASCEND ARANGE
[DEBUG] flag_gems.runtime._ascend.ops.cat.cat: GEMS_ASCEND CAT
[DEBUG] flag_gems.runtime._ascend.ops.embedding.forward: GEMS_ASCEND EMBEDDING FORWARD
[DEBUG] flag_gems.runtime._ascend.ops.index_select.index_select: GEMS_ASCEND INDEX SELECT
[DEBUG] flag_gems.runtime._ascend.ops.index.index: GEMS_ASCEND INDEX
[DEBUG] flag_gems.runtime._ascend.ops.argmax.argmax: GEMS_ASCEND ARGMAX
```

### Additional Context

- The issue appears to be specific to **huawei** platform and may not reproduce on standard CUDA (NVIDIA) environments.
- 5 operator(s) were identified through automated binary search / group testing.
- Disabling these operators restores acceptable accuracy/performance levels.

### Possible Directions

- Profile the identified operators to find performance bottlenecks
- Check Triton autotuning configurations for this platform
- Compare kernel launch overhead with native implementations
